packages/e2e-test-utils/src/get-all-block-inserter-item-titles.js

```diff
@@ -20,5 +20,5 @@ export async function getAllBlockInserterItemTitles() {
 			return inserterItem.innerText;
 		} );
 	} );
-	return [ ...new Set( inserterItemTitles ) ].sort();
+	return [ ...new Set( inserterItemTitles ) ];
```

Comment:
I changed this in https://github.com/WordPress/gutenberg/pull/50510/commits/8cb75c86b57bb06360fc279cf21897c18053bd53.

Because it does sorting on the results the tests cannot assert on the priority of the blocks.

This helper is "doing it wrong". Instead it should return raw results and the _consumer_ should do the sorting.

Let's see which tests break before deciding how to proceed from here.

Reply:
Do we think this should be it's own sub PR?